lib/effectiveCallNumber.js

```diff
@@ -14,23 +18,32 @@ import { get } from 'lodash';
  *   <EffectiveCopy>
  *
  * @param {object} item an item record as returned from /inventory/items/${item-id}
+ * @param {object} item returned from circulation/loans with structure {item: {}}
  * @return {string} the effective call number
  */
 export default function effectiveCallNumber(item) {
-  const prefix = get(item, 'effectiveCallNumberComponents.prefix') ||
-    get(item, 'callNumberComponents.prefix', '');
-  const suffix = get(item, 'effectiveCallNumberComponents.suffix') ||
-    get(item, 'callNumberComponents.suffix', '');
-  const callNumber = get(item, 'effectiveCallNumberComponents.callNumber') ||
-    get(item, 'callNumberComponents.callNumber', '');
+  const rootItem = item?.item ?? item;
 
-  return [
+  const prefix = get(rootItem, 'effectiveCallNumberComponents.prefix') ||
+    get(rootItem, 'callNumberComponents.prefix', '');
+  const suffix = get(rootItem, 'effectiveCallNumberComponents.suffix') ||
+    get(rootItem, 'callNumberComponents.suffix', '');
+  const callNumber = get(rootItem, 'effectiveCallNumberComponents.callNumber') ||
+    get(rootItem, 'callNumberComponents.callNumber', '');
+
+  const result = compact([
     prefix,
     callNumber,
     suffix,
-    get(item, 'volume', ''),
-    get(item, 'enumeration', ''),
-    get(item, 'chronology', ''),
-    get(item, 'copyNumbers[0]', ''),
-  ].join(' ');
+    get(rootItem, 'volume', ''),
+    get(rootItem, 'enumeration', ''),
+    get(rootItem, 'chronology', ''),
+    get(rootItem, 'copyNumbers[0]', ''),
+  ]);
+
+  if (!has(item, 'item')) {
+    result.push(get(rootItem, 'copyNumbers[0]', ''));
+  }
```

Comment:
This doesn't seem right: if `copyNumber` is present on `rootItem` and no `item.item` value is present, `copyNumber` will be duplicated.